SiteOutline.md

```diff
@@ -0,0 +1,119 @@
+
+* the Organizations section as exists now in production site homepage
+
+#### Presentations
+
+* the Presentations section as exists now in production site homepage
+
+#### Publications
+
+* the Publications section as exists now in production site homepage
+
+#### Tools
+
+* the Tools section as exists now in production site homepage
+
+#### Citations
+
+* the Citations section as exists now in production site homepage
+
+#### Resources
+
+* This is largely just a link to our contact information and a PDF download of the 2016 environmental scan. We may want to move this elsewhere in the new site, and eventually, have it give a link to the OSF repository download for the PDF.
+* Contact information is definitely elsewhere in the new site.
+
+## **Framework**:
+
+This will shift as the Framework work continues and outputs shift.
+
+For now, the top level page will have preliminary/introductory materials on the the purposes of the Framework, approaches to use, etc.
+
+The following will be... child pages? posts? embedded? other?
+
+#### Metadata assessment metrics (2016)
+
+* Relate to origin in discussions of “quality” and environmental scan
+* Also related to work on
+* Relate to discussions/review of other work - including Europeana QA
+
+#### Use cases for Metadata Assessment (2016)
+
+* Originally meant to inform metadata assessment framework, give areas to target
+* Responses / limitation thereof
+
+#### Metadata Application Profile Guidelines (2017)
+
+* Relate to origin in discussions of “quality” and environmental scan
+* Also related to work on
+* Relate to discussions/review of other work - including Europeana QA
+
+#### Framework for performing metadata assessment // generic checklist for guidance (2016-7)
+
+Should build off of above work and parallel work below
+
+## **Tools**: Expanded tools recommendations (2016-?)
+
+* Take work that fed into Environmental Scan and expand
+* Site page will embed or link to Clearinghouse search interface
+
+## **Profiles**: Metadata Profiles and Mappings Clearinghouse (2017)
+
+* Site page will embed or link to Clearinghouse search interface, links to the work, repo, or just description of the output as decided.
+
+## **Collaborations**: Collaborations with Other Groups (2017)
+
+* List of collaborations with other groups as projects emerge.
+* Way for groups to get involved if interested.
+
+## **Glossary**: Glossary & References (2016-7)
+
+* Glossary of terms used on the site.
+* Seems appropriate to exist outside of the Framework page(s).
+
+## **Contact & Get Involved**
+
+* Take and expand what exists already.
+* Might also be the right place to embed the update form.
+
+## **About**
+
+* Take and expand what exists already. May be wrapped up in the Introduction / Homepage section.
+
+## **Sidebar** with site structure
+
+* This will have to be regenerated elsewhere.
+* This is an includes bit, not a page...?
+* Is generated now based on Jekyll theme use of posts naming/metadata.
```

Comment:
And by simplify, I mean "Not having to list out all pages + child pages" on the navigation includes but perhaps instead some code to indicate what should display based on the page that you're on. That way we don't have to worry about adding new child pages as they're developed.